infrastructure/server-setup/group_vars/all.yml

```
@@ -14,3 +14,4 @@ backup_server_user: 'backup'
 backup_server_user_home: '/home/backup'
 crontab_user: root
 provisioning_user: provision
+single_node: true
```

```suggestion
single_node: false
```

I guess we agreed to set this variable false by default.

That will be good compromise why we need explicitly mention variable per environment.